protocol/bc/txinput.go

```diff
@@ -115,6 +117,10 @@ func (t *TxInput) Amount() uint64 {
 	return si.Amount
 }
 
+func (t *TxInput) VMVersion() uint64 {
+	return t.TypedInput.vmVersion()
+}
+
 func (t *TxInput) ControlProgram() []byte {
```

Comment:
I just noticed TxInput.ControlProgram and TxInput.IssuanceProgram are no longer called anywhere. We could potentially remove them in this patch.